components/Feature/SharePlan/index.test.js

```diff
@@ -68,4 +68,18 @@ describe('Share plan', () => {
       getByText('Something went wrong. The plan could not be shared.')
     ).toBeInTheDocument();
   });
+
+  it('renders disabled checkbox when phone numbers and emails do not exist ', () => {
+    const plan = {
+      firstName: 'Sally',
+      lastName: 'West',
+      numbers: [],
+      emails: []
+    };
+    const { getByLabelText } = render(<SharePlan plan={plan} />);
+
+    expect(getByLabelText('No numbers found.').disabled).toBe(true);
+    expect(getByLabelText('No emails found.')).toBeInTheDocument();
```

Comment:
Should these two lines be the same(ish)?

Reply:
that's a good point. I think the reason they were different is because amt email will always be disabled, but it's worth changing for there future 👌